packages/plugin-core/src/views/LookupView.ts

```diff
@@ -127,6 +129,23 @@ export class LookupView implements vscode.WebviewViewProvider {
     }
   }
 
+  public refreshLookupView(opts: { buttons: DendronBtn[] }) {
+    const { buttons } = opts;
+    const payload: LookupModifierStatePayload = buttons.map(
+      (button: DendronBtn) => {
+        return {
+          type: button.type,
+          pressed: button.pressed,
+        };
+      }
+    );
+
+    const lookupView = this._extension.getTreeView(
+      DendronTreeViewKey.LOOKUP_VIEW
+    ) as LookupView;
+    lookupView.refresh(payload);
+  }
```

Comment:
This probably can be merged with the `refresh` method below and simplified: 

```suggestion
  public refresh(opts: { buttons: DendronBtn[] }) {
    const { buttons } = opts;
    const payload: LookupModifierStatePayload = buttons.map(
      (button: DendronBtn) => {
        return {
          type: button.type,
          pressed: button.pressed,
        };
      }
    );

    if (this._view) {
      this._view.webview.postMessage({
        type: LookupViewMessageEnum.onUpdate,
        data: { payload },
        source: "vscode",
      });
    }
  }
```

Reply:
makes sense, thanks for pointing out